docs/release-notes/NuGet-5.6.md

```diff
@@ -0,0 +1,79 @@
+---
+title: NuGet 5.6 Release Notes
+description: Release notes for NuGet 5.6 including new features, bug fixes, and DCRs.
+author: chgill-msft
+ms.author: chgill
+ms.date: 05/19/2020
+ms.topic: conceptual
+---
+
+# NuGet 5.6 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version| Available in .NET SDK(s)|
+|:---|:---|:---|
+| [**5.6.0**](https://nuget.org/downloads) | [Visual Studio 2019 version 16.6](https://visualstudio.microsoft.com/downloads/) | [3.1.300](https://dotnet.microsoft.com/download/dotnet-core/3.1)<sup>1</sup> |
+
+<sup>1</sup>Installed with Visual Studio 2019 with .NET Core workload
+
+## Summary: What's New in 5.6
+
+* Support pre-release packages with floating versions. Version="*-*" and similar. - [#912](https://github.com/NuGet/Home/issues/912)
+
+### Issues fixed in this release
+
+**Bug:**
+
+* `nuget push *.nupkg` fails when snupkg does not exist - [#8148](https://github.com/NuGet/Home/issues/8148)
+
+
+* Pack, and several other code paths, fails dependant on locale. Use RegexOptions.CultureInvariant - [#8246](https://github.com/NuGet/Home/issues/8246)
+
+* Perf: DG Spec for unloaded project scenarios should not be written in preview restores - [#8793](https://github.com/NuGet/Home/issues/8793)
+
+* Restore:  improve performance by caching solution dependency graph spec - [#9201](https://github.com/NuGet/Home/issues/9201)
+
+* PM UI doesn't work for sdk style projects after installing a package with PMConsole - [#9203](https://github.com/NuGet/Home/issues/9203)
+
+* [Test Failure] Embedded icon can’t be shown in PM UI with local package feed - depending on / vs \ - [#9225](https://github.com/NuGet/Home/issues/9225)
+
+* NuGetVersion.TryParseStrict() should return false if it fails to parse - [#9255](https://github.com/NuGet/Home/issues/9255)
+
+* `nuget.exe push` help for `-source`, should be source name, not source uri - [#9265](https://github.com/NuGet/Home/issues/9265)
+
+* `dotnet nuget add package SourceUri`  creates bad default package source name - [#9277](https://github.com/NuGet/Home/issues/9277)
+
+* Screen reader doesn't announces "Searching..." message when switching tabs - [#9307](https://github.com/NuGet/Home/issues/9307)
+
+* [Test Failure] Focus-rect color in themes for PM UI tabs - [#9336](https://github.com/NuGet/Home/issues/9336)
+
+* Several accessibility bugs reported by users - [#9393](https://github.com/NuGet/Home/issues/9393)
+

```

Comment:
In the [previous release notes](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master/docs/release-notes/NuGet-5.5.md), accessibility fixes are mentioned in the `Summary` section.

Reply:
turns out this issue didn't have a fix in this release. so we'll remove.

generally, PM and somebody from engineering (used to be Karan and I each release) discuss and figure out which things to highlight...usually features and any extra effort in an area (like accessibility in 5.5)